### PR TITLE
feat(tags): Display tags in masterclass cards [WD-6402]

### PR DESCRIPTION
**Pull request by aaryanporwal**
# Done

- [x] Tags that are applied on the video, now show up in the masterclass cards. 
- [x] Move description below topic, for better visibility, before it was pushed at the bottom of the card.

### CODE REVIEW

**Review by lorumic — COMMENTED** [review]
Code looks good, but I would suggest to bottom-align the footer, so that when we have cards with a different height in the same row, the section that takes the white space is the one in the middle, not the footer. I think it would look better, and it is also the chosen approach for u.com/blog.

See one example row where this is particularly evident:

image

File: templates/shared/_card--masterclass.html [hunk 1]
```diff
@@ -8,9 +8,11 @@
     {% endif %}
   </a>
   <div class="p-card__inner">
-    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
-    <p class="u-text--muted u-no-margin--bottom">Duration: {{ session.Duration }}</p>
+    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a>
+    </h3>
     <p>{{ session.Description }}</p>
+    <p class="u-text--muted u-no-margin--bottom">{{ session.Tags }}</p>
+    <p class="u-text--muted u-no-margin--bottom">Duration: {{ session.Duration }}</p>
```
Comment (edlerd) [inline templates/shared/_card--masterclass.html:15]: Shall we remove the duration? It is not accurate and probably not so important when browsing the videos, wdyt?

Comment (edlerd) [reply templates/shared/_card--masterclass.html:15]: Or keep it but remove the "duration:" prefix

Comment (aaryanporwal) [reply templates/shared/_card--masterclass.html:15]: I think it’s good to have Duration there. In future we can add a filter by duration as well. And yes we can get rid of “Duration: “ prefix for sure!

**Comment by aaryanporwal** [comment]
@lorumic  Yes! That’s a great thing to have. I’ll work on it.

File: templates/shared/_card--masterclass.html [hunk 2]
```diff
@@ -8,9 +8,11 @@
     {% endif %}
   </a>
   <div class="p-card__inner">
-    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
-    <p class="u-text--muted u-no-margin--bottom">Duration: {{ session.Duration }}</p>
+    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a>
+    </h3>
     <p>{{ session.Description }}</p>
+    <p class="u-text--muted u-no-margin--bottom">{{ session.Tags }}</p>
```
Comment (lorumic) [inline templates/shared/_card--masterclass.html:14]: Another idea to make this nicer: what about splitting it on the comma to have a list, and use chips for each tag?

Comment (edlerd) [reply templates/shared/_card--masterclass.html:14]: I think the muted text is better suited -- tags would be quite eye-catchey.

Comment (lorumic) [reply templates/shared/_card--masterclass.html:14]: Tags are meant to be so. And we already have muted text in the line below for the duration, so it's hard to distinguish them. Maybe ask input to a designer?

Comment (aaryanporwal) [reply templates/shared/_card--masterclass.html:14]: @lorumic I had tried putting the chips in there, and @edlerd is right, it was taking a lot of space.
Check the screenshot:
screenshot

Comment (aaryanporwal) [reply templates/shared/_card--masterclass.html:14]: This is screenshot with dense chips:
screenshot 2

Comment (lorumic) [reply templates/shared/_card--masterclass.html:14]: I don't see anything wrong except that there should be more spacing between the chips and the duration line.

Comment (aaryanporwal) [reply templates/shared/_card--masterclass.html:14]: After the discussion in our standups, we'll go ahead and use the dense chips.

Comment (aaryanporwal) [reply templates/shared/_card--masterclass.html:14]: We've decided to go in favor of muted text for now, because chips are meant to be used interactively.

**Review by edlerd — APPROVED** [review]
LGTM, thanks for the changes.

**Pull request merged**